brain-marks/brain_marksApp.swift

```diff
@@ -12,15 +12,17 @@ import TelemetryClient
 
 @main
 struct brain_marksApp: App {
-    
+    @AppStorage("migrationToCoreDataRan")
```

Comment:
what does adding this variable here do, when it isn't used?

Reply:
the purpose of this being here is to run before the migration service does on first launch. It'll default to false on the first app run since being introduced. It's then set to true to not run the migration again.